### PR TITLE
Updated texts for english and swedish

### PR DESCRIPTION
**Pull request by KentMossback**

### CODE REVIEW

File: apps/pxweb2/public/locales/sv/translation.json [hunk 1]
```diff
@@ -9,48 +9,161 @@
...
+        "file": {
+          "title": "Spara som fil",
+          "excel": "Excel (xlsx)"
+        },
+        "imagefile": {
+          "title": "Spara som diagram",
+          "png": "Diagram (png)"
+        },
+        "link": {
+          "title": "Save as länk",
```
Comment (MikaelNordberg) [inline apps/pxweb2/public/locales/sv/translation.json:130]: Not correct swedish

File: apps/pxweb2/public/locales/sv/translation.json [hunk 2]
```diff
@@ -9,48 +9,161 @@
...
+          "title": "API",
+          "description": "Description text..."
+        }
+      },
+      "help": {
+        "title": "Hjälp"
+      }
+    },
+    "main_content": {
+      "last_updated": "Last updated",
```
Comment (MikaelNordberg) [inline apps/pxweb2/public/locales/sv/translation.json:143]: Not swedish

File: apps/pxweb2/public/locales/sv/translation.json [hunk 3]
```diff
@@ -9,48 +9,161 @@
...
+          "title": "Anpassa",
+          "pivot": {
+            "title": "Pivotera"
+          },
+          "rearrange": {
+            "title": "Anpassa table",
+            "description": "Beskrivande text..."
+          },
+          "change_order": {
+            "title": "Change order",
```
Comment (MikaelNordberg) [inline apps/pxweb2/public/locales/sv/translation.json:104]: Not swedish

File: apps/pxweb2/public/locales/en/translation.json [hunk 4]
```diff
@@ -81,7 +81,7 @@
         }
       },
       "view": {
-        "title": "View",
+        "title": "Distplay",
```
Comment (MikaelNordberg) [inline apps/pxweb2/public/locales/en/translation.json:84]: Not correct english

File: apps/pxweb2/src/@types/resources.d.ts [hunk 5]
```diff
@@ -82,7 +82,7 @@ interface Resources {
           }
         },
         "view": {
-          "title": "View",
+          "title": "Distplay",
```
Comment (MikaelNordberg) [inline apps/pxweb2/src/@types/resources.d.ts:85]: Not correct english

**Review by MikaelNordberg — CHANGES_REQUESTED** [review]
Found some misspellings and some sentences that should be translated.

File: apps/pxweb2/public/locales/en/translation.json (hunk 4 above)
Comment (KentMossback) [reply apps/pxweb2/public/locales/en/translation.json:84]: Is it "View" or "Display" that is incorrect?

File: apps/pxweb2/public/locales/en/translation.json [hunk 6]
```diff
@@ -81,7 +81,7 @@
         }
       },
       "view": {
-        "title": "View",
+        "title": "Distlay",
```
Comment (MikaelNordberg) [inline apps/pxweb2/public/locales/en/translation.json:84]: Not correct english

**Review by MikaelNordberg — CHANGES_REQUESTED** [review]
One word need to be fixed

**Review by MikaelNordberg — DISMISSED** [review]
Looks good

**Review by MikaelNordberg — DISMISSED** [review]
Looks good

**Review by MikaelNordberg — APPROVED** [review]
Looks good

**Pull request merged**